startup_heroku.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+if [ -z "$PR" ];
+then
+  echo "Usage: PR=<pull-request-number> ./startup_heroku.sh"
+  exit 1
+fi
+
+export APP_NAME="calculators-pr-$PR"
+export HEROKU_REMOTE="heroku-$APP_NAME"
+
+echo
+echo "# Creating a new Heroku App"
+echo
+heroku apps:create --region eu --remote $HEROKU_REMOTE $APP_NAME
+
+echo
+echo "# Configuring Heroku ready for Smart Answers"
+echo
+heroku config:set \
+--app $APP_NAME \
+GOVUK_APP_DOMAIN=integration.publishing.service.gov.uk \
+PLEK_SERVICE_CONTENTAPI_URI=https://www.gov.uk/api \
+PLEK_SERVICE_STATIC_URI=https://assets-origin.integration.publishing.service.gov.uk/ \
+RUNNING_ON_HEROKU=true \
+EXPOSE_GOVSPEAK=true \
```

Comment:
I don't think `EXPOSE_GOVSPEAK` is used in the Calculators app i.e. I think it is specific to Smart Answers.